plugins/tekton/src/components/pipeline-topology/PipelineVisualization.test.tsx

```diff
@@ -42,7 +48,40 @@ describe('PipelineVisualization', () => {
       screen.getByText(mockKubernetesPlrResponse.pipelineruns[1].metadata.name),
     ).toBeInTheDocument();
 
-    expect(queryByTestId('pipeline-visualization')).not.toBeNull();
+    expect(queryByTestId('latest-pipelineRun-visualization')).not.toBeNull();
+  });
+
+  it('should render the pipeline run visualization when pipelineRun name exists in the params', async () => {
```

Comment:
this doesn't need to be async

```suggestion
  it('should render the pipeline run visualization when pipelineRun name exists in the params', () => {
```